dev_guide/integrating_external_services.adoc

```diff
@@ -120,66 +118,78 @@ be] loopback (127.0.0.0/8), link-local (169.254.0.0/16), or link-local multicast
 value in the service defined in the previous step.
 
 [[mysql-define-service-using-fqdn]]
-==== Using FQDN
-Use an `ExternalName` service without defining any ports or endpoints.
+==== Using an External Domain Name
+
+Using external domain names make it easier to manage an external service
+linkage, because you don't have to worry about the external service's IP
```

Comment:
Would it be more clear if we change `about the external service's IP addresses changing.` to `about changing IP addresses of the external service.`?

Reply:
I think that does change the meaning. Might imply that the external service's IP change by design, which I don't think is accurate.